gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -50,4 +53,21 @@ else
     trap 'docker kill {genconf_tar}' HUP QUIT INT TERM
     docker run --rm --name=$DCOS_INSTALLER_CONTAINER_NAME -i -p $PORT:9000 -v $(pwd)/genconf/:/genconf {docker_image_name} "$@"
 fi
-exit $?
+
+if [ $? -ne 0 ]; then
+    echo "Problems running installer"
+    exit $?
+fi
+
+if [ -d genconf/ca ]; then
+    psk=$(cat genconf/ca/psk 2>/dev/null || echo "")
+    if [ -z "$psk" ]; then
+        echo "WARNING: Bootstrap CA service PSK is unset"
+    fi
+
+    echo "Starting bootstrap CA service in the background"
+    genconf/bin/dcoscertstrap -d genconf/ca/ serve --address :443 --psk "$psk" &
```

Comment:
Let's make the address explicit here to make the reader not looking it up in the dcoscertstrap code.

Reply:
We should also listen on a sub-path of the endpoint: e.g. `https://bootstrap:443/ca/`. This will keep open the option to use the secure HTTPS endpoint for other services in the future.

Reply:
The service listens on TLS endpoint only

Reply:
This would also require a change to `dcos-bootstrap-ca` to use / support listening on a sub-path.

In the future we may want to support other services on the bootstrap node, and particularly on the 443 endpoint. At that point, we may add a "bootstrap Admin Router" to point `ca` requests to `dcos-bootstrap-ca` and another path to a different service.

So we can use `/ca` now to give us that future flexibility without needing to change this client later.